UnityProject/Assets/Scripts/Items/Tool/AdvancedLightReplacer.cs

```diff
@@ -84,9 +84,11 @@ private void AddLightToFixture(ItemSlot target, LightSource source, HandApply in
 		[TargetRpc]
 		private void LightTunerWindowOpen(NetworkConnection target)
 		{
+			UIManager.Instance.GlobalColorPicker.CurrentColor = currentColor;
 			UIManager.Instance.GlobalColorPicker.EnablePicker(SetColorToTune);
 		}
 
+		[Command(requiresAuthority = false)]
```

Comment:
could you Put validation checks in this,  I think if you put a field for connection it automatically pick it up which client requested it, then it's easy to validate if they contract with the item

Reply:
I don't think this is a big security problem to require validation

Reply:
I think it's always good practice